### DISCUSSION TITLE
How to Specify the player's translation

### DISCUSSION THREAD

**Discussion by Birditch** [KyoriPowered/adventure#908] [General]
How do I specify the player's language when I send a message to them? Specify and then send the specified translation.
I have found that some players will exist who do not set their client language to their native language. And most will use English, but in that case, when we use Audiences to send Component messages to them, they will send messages translated into their client language. This is not good.
So, I would like to know, how to specify the translation for Audiences?

**Comment by kezz** [comment]
Translations using translatable components are done client-side. You cannot override this as the translation is done when the message has been received by the client.

If you are talking about server side translations using the global translator, these are done automatically by most native implementations and platform builds. These platforms may provide a way to overwrite or set the locale of the audience member, but that is not guaranteed or part of the Adventure API. In this case you could manually run the components through the global translator before sending them to the audiences, in which case you can specify the locale.
